.babelrc

```diff
@@ -1,4 +1,4 @@
 {
   "plugins": ["transform-class-properties"],
-  "presets": ["env", "react"]
+  "presets": ["@babel/env", "@babel/react"]
```

Comment:
nice :)

Reply:
yeah, that's because of the new version of babel